Inital Design

pskinnunen opened the pull request:
Made the changes as suggested and found a couple items I missed as well in the output formatting.

pskinnunen committed:
Inital Design

jwmarcus commented:
We have a few things we could improve here.

- Spelling: Please go through this again, there are many spelling errors.
- Formatting: PEP8 found [here](https://www.python.org/dev/peps/pep-0008/) establishes style suggestions for things like lists. See [this section](https://www.python.org/dev/peps/pep-0008/#code-lay-out) for tips on lists.
- Invocation: Many Python programs utilize built-in variables when running as a stand-alone script. Check out [this documentation](https://docs.python.org/3/library/__main__.html) to learn how to use top-level script environment variables for this script.

pskinnunen committed:
Corrected spelling errors
Adjusted formating
Added __main__ referencing for running the script.
Corrected output formating.

jwmarcus commented:
Text and formatting look MUCH better. Great use of the double colon trick, as explained [at this link](https://stackoverflow.com/questions/3453085/what-is-double-colon-in-python-when-subscripting-sequences). I'm going to pull this into master.

jwmarcus commented:
Don't mind me accidentally closing that branch, I meant to merge it.
